Source/WebCore/platform/network/curl/CurlMultipartHandleClient.h

```diff
@@ -27,13 +27,11 @@
 
 namespace WebCore {
 
-class CurlResponse;
-class FragmentedSharedBuffer;
-
 class CurlMultipartHandleClient {
 public:
-    virtual void didReceiveHeaderFromMultipart(const Vector<String>&) = 0;
+    virtual void didReceiveHeaderFromMultipart(Vector<String>&&) = 0;
     virtual void didReceiveDataFromMultipart(const SharedBuffer&) = 0;
```

Comment:
The argument should be `SharedBuffer&&`.

Reply:
Fixed with Ref<SharedBuffer>&&.